client/my-sites/guided-transfer/guided-transfer.jsx

```diff
@@ -7,23 +7,64 @@ import page from 'page';
 /**
  * Internal dependencies
  */
+import Button from 'components/button';
 import HeaderCake from 'components/header-cake';
+import AccountInfo from './account-info';
 
 export default React.createClass( {
 	displayName: 'GuidedTransfer',
 
 	propTypes: {
+		hostSlug: PropTypes.string,
 		siteSlug: PropTypes.string.isRequired
 	},
 
 	showExporter() {
 		page( `/settings/export/${this.props.siteSlug}` );
 	},
 
+	showHostSelection() {
+		page( `/settings/export/${this.props.siteSlug}/guided` );
+	},
+
+	showBluehostAccountInfo() {
+		page( `/settings/export/${this.props.siteSlug}/guided/bluehost` );
+	},
+
+	goBack() {
+		if ( this.props.hostSlug ) {
+			this.showHostSelection();
+		} else {
+			this.showExporter();
+		}
+	},
+
+	getHostInfo( host ) {
+		switch ( host ) {
+			case 'bluehost': return {
+				label: this.translate( 'Bluehost' ),
+				url: '#bluehost'
+			};
+		}
```

Comment:
@dmsnell showed me a nice trick to simplify these sorts of switch cases: we can use lodash's `get` function:

```js
get(object, path, [defaultValue])
```

eg:

```js
import { get } from 'lodash/get';

const hosts = {
	'bluehost': {
		label: this.translate( 'Bluehost' ),
		url: '#bluehost',
	},
}

return get( hosts, host );
```